app/views/__tests__/Import.spec.js

```diff
@@ -1,13 +1,51 @@
-jest.spyOn(console, 'log').mockImplementationOnce(() => {});
-
+import { fireEvent, render, NativeTestEvent, act } from '@testing-library/react-native'
 import 'react-native';
+import {Linking} from 'react-native';
 import React from 'react';
 import renderer from 'react-test-renderer';
 import Import from '../Import';
 
-it('renders correctly', () => {
-  const tree = renderer
-    .create(<Import />)
+jest.spyOn(console, 'log').mockImplementation(() => {});
```

Comment:
Does this act globally?

I think these `jest.spyOn()` should be in a `beforeAll()` or `beforeEach` block

Reply:
looks like this is applied only within the context of this file. [more details in .resetModules() docs](https://jestjs.io/docs/en/configuration#resetmodules-boolean)